AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -19,6 +19,7 @@ love.mail.ru###adblock-in-browser
 ! END: mail.ru
 !
 ! Yandex
+yandex.ru,yandex.ua,yandex.by,yandex.kz#%#//scriptlet("abort-on-property-read", "Object.prototype.RtbManagerStart")
```

Comment:
Is the rule below required?

Reply:
I guess no, can't find `_waitAdvManagerAndRender` in their scripts.